live-images: remove outdated flavor image info and simplify (#637)
void-linux/void-docs: src/installation/live-images/index.md

# Live Installers

Void provides live images containing a base set of utilities useful for

> installing and/or repairing systems. Images can be found on the

Reviewer: "in the mirrors", I think?

Author: I am not an English major, but I think on is appropriate here?

[mirrors](../../xbps/repositories/mirrors/index.md) for either `glibc` or `musl`

> systems on `x86_64`, and for `glibc` only on `i686`. Users of other

Reviewer: ```suggestion
on `x86_64` systems, and for `glibc` on `i686` ones. Users of other
```

architectures will need to perform a [manual installation](../guides/index.md).

## Base Image

> The base images contain just enough for a usable Void system, with utilities for
> basic hardware and network configuration included. Packages can be installed

Reviewer 2: ```suggestion
The base image contains just enough packages to install a usable Void system, with utilities for
basic hardware and network configuration included. Packages can be installed
```

the `with...` sounds a bit weird to me

> from the image offline, or by downloading the latest versions online.

Reviewer: This isn't clear that it means installing packages onto a new install.

Reviewer: Oh, I see paper had noticed this already!

## XFCE Desktop Image

> Void also offers an XFCE image. The live environment and installed system
> include a desktop environment, web browser, and basic applications
> pre-configured, but the installation procedure is the same. Void must be

Reviewer 2: ```suggestion
Void also offers an XFCE image. The live environment
includes a desktop environment, web browser and basic desktop applications,
but the installation procedure is the same. Void must be
```

# Live Installers

Void provides live images containing a base set of utilities useful for
installing and/or repairing systems. Images can be found on the

> [mirrors](../../xbps/repositories/mirrors/index.md) for either `glibc` or `musl`
> systems on `x86_64` systems, and for `glibc` on `i686` ones. Users of other

Reviewer 2: ```suggestion
[mirrors](../../xbps/repositories/mirrors/index.md) with either `glibc` or `musl`
on `x86_64` systems and only with `glibc` on `i686` ones. Users of other
```

architectures will need to perform a [manual installation](../guides/index.md).

## Base Image

> The base image contains just enough for a usable Void system, with utilities for

Reviewer 2: ```suggestion
The base image contains just enough tools for installing a usable Void system, with utilities for
```

Author: I think "includes just enough tools to install a usable Void system" works even better?

basic hardware and network configuration included. The installation can be

> performed online, which will download the latest package versions versions, or

Reviewer 2: ```suggestion
performed online, which will download the latest package versions, or
```

offline, which will use the versions of packages available on the installation
medium.

## XFCE Desktop Image

Void also offers an XFCE image. The live environment includes a desktop

> environment, web browser, and basic desktop applications, but the installation

Reviewer 2: ```suggestion
environment, web browser and basic desktop applications, but the installation
```
at least I think

Author: Is Oxford comma against the style guide?

> procedure is the same. Choose the `Local` package source in void-installer to

Reviewer 2: ```suggestion
procedure is the same as with the base image. Choose the `Local` package source in void-installer to
```
